tools/cmgen/src/cmgen.cpp

```diff
@@ -933,7 +933,7 @@ void iblRoughnessPrefilter(
         .endianness = KtxBundle::ENDIAN_DEFAULT,
         .glType = KtxBundle::R11F_G11F_B10F,
         .glTypeSize = 1,
-        .glFormat = KtxBundle::R11F_G11F_B10F,
```

Comment:
Why was this removed? We need this format for our IBLs. It breaks all environment lighting.

Reply:
We're still using the 11_11_10 (see the `glType` field), this needed to be unsized to be valid KTX.  Is stuff breaking for you?

Reply:
Actually the issue isn't here. The issue is above.